views/editor/warnings.html

```diff
@@ -0,0 +1,17 @@
+{% block warnings %}
+
+<div id="browser-support-warning" class="hide">
+  <div class="message-box">
+    <h1>{{ gettext("errorUnsupportedBrowser") }}</h1>
+    <p>
+      {{ gettext("errorUnsupportedBrowserSuggestion") | instantiate | safe }}
+    </p>
+    <div title="{{ gettext("errorUnsupportedBrowserIgnoreTitle") }}" class="let-me-in">{{ gettext("errorUnsupportedBrowserIgnore") }}</div>
+  </div>
+</div>
+
+<div class="top-alert-bar hide" id="serviceworker-warning">
+  {{ gettext("refreshForUpdates") | safe }}
```

Comment:
I think you should do a separate string vs. trying to reuse this logic around having that event show this.  If they hit this error screen, they need to try a refresh, basically.  We should give them a button to press that says "Refresh" to download updates and try again.